src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -65,7 +62,8 @@ function bv_adios2_info
     export ADIOS2_FILE=${ADIOS2_FILE:-"adios2-${ADIOS2_VERSION}.tar.gz"}
     export ADIOS2_COMPATIBILITY_VERSION=${ADIOS2_COMPATIBILITY_VERSION:-"${ADIOS2_VERSION}"}
     export ADIOS2_URL=${ADIOS2_URL:-"https://github.com/ornladios/ADIOS2/archive/v2.5.0"}
-    export ADIOS2_BUILD_DIR=${ADIOS2_BUILD_DIR:-"ADIOS2-"${ADIOS2_VERSION}}
+    export ADIOS2_SRC_DIR=${ADIOS2_SRC_DIR:-"ADIOS2-"${ADIOS2_VERSION}}
+    export ADIOS2_BUILD_DIR=${ADIOS2_BUILD_DIR:-"${ADIOS2_SRC_DIR}-build-ser"}
```

Comment:
Just coursious, why spell out BUILD but abbreviate SOURCE to SRC?

Reply:
Cause SRC was already being used.